config_defaults.yml

```diff
@@ -30,7 +30,7 @@ Gemfile:
 Rakefile:
   default_disabled_lint_checks:
   - 'relative'
-  - 'disable_80chars'
+  - 'disable_char_check'
```

Comment:
Is char_check in previous puppet-lint versions?

Also note that puppetlabs-spec-helper already sets up default exclusions

`'80chars',
    'class_inherits_from_params_class',
    'class_parameter_defaults',
    'documentation',
    'single_quote_string_with_variables']`

https://github.com/puppetlabs/puppetlabs_spec_helper/blob/master/lib/puppetlabs_spec_helper/rake_tasks.rb#L384-L391